interface/procedure_tools/labcorp/ereq_form.php

```diff
@@ -365,25 +375,27 @@ function ereqForm($pid, $encounter, $form_id, $reqStr = null, $doDoc = true)
         $pdfContent .= '<td style="padding-left:8px;" colspan="8"><b>Diagnosis Codes:</b>List all applicable Diagnosis codes. Must be at Highest Level Specificity.</td>';
         $pdfContent .= '</tr>';
         $pdfContent .= '<tr>';
-        $pdfContent .= '<td style="width:12.5%;" >' . text(str_replace('ICD10:', '', $all_diagnoses[0])) . '</td>';
-        $pdfContent .= '<td style="width:12.5%;" >' . text(str_replace('ICD10:', '', $all_diagnoses[1])) . '</td>';
```

Comment:
Yes this was expecting ICD10 which are required in good old USA. I think code type was removed for space on the requisition form.
BTW: I didn't design this tedious form but inherited it from I think was Nilesh for Jit whom donated to openemr.
I just made it work.